### ISSUE TITLE
[Suggestion] Switch WASD to pitch/yaw in Boost/Light-Cruise modes

### ISSUE THREAD

**Issue by Asnro** [tsunamayo/Starship-EVO#2984]
This would allow starship flight the option to be entirely left-handed, leaving mouse for aiming turrets/gimbals or otherwise free during extended flights. I believe this would improve flight ergonomics and user experience, especially for players not experienced in mouse-controlled flight. This is already the case with hovercraft (A/D to yaw), which have proven quite user-friendly.

**Comment by Gliese-832-c** [comment]
If this gets implement, as with everything related to control and gameplay, please make it a configurable option. I prefer the more precise control that the mouse gives you.

**Comment by Asnro** [comment]
I guess I should have clarified that this wouldn't replace mouse control, but give additional key controls. Currently in freelook+boost mode a ship is impossible to control aside from roll, this would allow that to be changed.

**Comment by KoPalKo** [comment]
For me, as a new player, that sounds pretty intuitive! Its a minimalistic and nice suggestion to increase the functionality of the turrets, i would like to see it implemented.

**Comment by tsunamayo** [comment]
Hum, I dont like making the scheme more complicated. Also people like to park their ship with the freelook mode, and translation is what you need for parking...
Cheers

**Comment by Asnro** [comment]
This change would not affect parking or translation, only Boost mode (Press Shift) and Light-Cruise mode (Press L) in which translation does not exist and WASD keys are useless.

Adding alternate controls to allow fully one-handed ship control (Regular mode for translating, parking, etc with WASD used for translation, Boost mode for "normal" forward flight with WASD now used for attitude control alongside normal mouse control) may be more complex to program but in my opinion would give a greater "comfort zone" to players.

**Comment by tsunamayo** [comment]
I see, I didnt look at the title enough ;). As it is more a power user thing how about mapping these to your arrows key or something else as secondary keys?

**Comment by Gliese-832-c** [comment]
Yes. The important thing here, especially since we are talking about probably the most important mechanic in a game, the controls, is to make it configurable. Give the user options, a lot of options, and ways to change those, and make them decide, how they want to play the game, themselves.
